packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/RecentSubjects.stories.js

```diff
@@ -105,56 +140,35 @@ export default {
   component: RecentSubjects,
   args: {
     dark: false
```

Comment:
I think this `args.dark` needs to be removed. I'm seeing an unusable toggle for 'dark' under Controls, for all Stories under Screens -> Project Home -> Recent Subjects -> *

(Not to be confused with Screens -> Classify -> Recent Subjects -> * , which look fine)